File: output/LLaMa/codenet/Python/Java/s895229833.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        int[] H = new int[N];
        for (int i = 0; i < N; i++) {
            H[i] = scanner.nextInt();
        }
        scanner.close();

        int countBest = 0;
        int countTmp = 0;

        for (int i = 0; i < N - 1; i++) {
            if (H[i] >= H[i + 1]) {
                countTmp++;
            } else {
                if (countTmp > countBest) {
                    countBest = countTmp;
                }
                countTmp = 0;
            }
        }

        if (countTmp > countBest) {
            countBest = countTmp;
        }

        System.out.println(countBest);
    }
}

File: dataset_reduzido/codenet/Python/Code/s895229833.py
N = int(input())
H = list(map(int, list(input().split())))

count_best, count_tmp = 0, 0

for i in range(N-1):                                                                                                                                                                                        
    if H[i] >= H[i+1]:                                                                                                                                                                                      
        count_tmp += 1                                                                                                                                                                                      
    else:                                                                                                                                                                                                   
        if count_tmp > count_best:                                                                                                                                                                          
            count_best = count_tmp                                                                                                                                                                          
                                                                                                                                                                                                            
        count_tmp = 0                                                                                                                                                                                       

if count_tmp > count_best:                                                                                                                                                                                  
    count_best = count_tmp                                                                                                                                                                                  

print(count_best)